geode-protobuf/src/main/java/org/apache/geode/protocol/protobuf/operations/GetRequestOperationHandler.java

```diff
@@ -42,6 +46,7 @@
     String regionName = request.getRegionName();
     Region region = executionContext.getCache().getRegion(regionName);
     if (region == null) {
+      logger.debug("Received Get request for non-existing region ");
```

Comment:
In other places you logged the region name.  Do you want to do that here as well?